engines/query-sparql/Dockerfile

```
@@ -1,23 +1,24 @@
 FROM node:18
 
+USER node
+
 # Install location
-ENV dir /var/www/@comunica/query-sparql/
+WORKDIR /var/www/@comunica/query-sparql/
 
 # Copy the engine files (generated from package.json!files)
-COPY config ${dir}/config/
-COPY lib/*.js ${dir}/lib/
-COPY bin/*.js ${dir}/bin/
-COPY engine-default.js package.json ${dir}
+COPY --chown=node:node config config/
+COPY --chown=node:node lib/*.js lib/
+COPY --chown=node:node bin/*.js bin/
+COPY --chown=node:node engine-default.js package.json ./
 
 # Set the npm registry
 ARG NPM_REGISTRY=https://registry.npmjs.org/
 RUN npm config set @comunica:registry $NPM_REGISTRY
 
 # Install the node module
-RUN cd ${dir} && npm install --only=production
+RUN npm install --only=production
```

npm WARN config only Use `--omit=dev` to omit dev dependencies from the install.

```suggestion
RUN npm install --omit=dev
```